config/sync/views.view.author_content.yml

```diff
@@ -0,0 +1,247 @@
+uuid: 54da7072-e790-4c1a-9dc1-cbbe730d4e3b
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.teaser
+    - node.type.article
```

Comment:
The article that's displayed should be look the same as the recipes (image on top content at the bottom) like the content spotlight